src/software/ai/hl/stp/stp_tactic_assignment_test.cpp

```diff
@@ -1,677 +0,0 @@
-}
-
-TEST_F(STPTacticAssignmentTest,
-       test_assigning_multiple_robots_to_goalie_tactic_goalie_set_on_team)
-{
-    // Test that only the robot set as the goalie on the team is assigned to the
-    // goalie tactic
-
-    // Put two robots right in front of the friendly goal
-    Team friendly_team(Duration::fromSeconds(0));
-    Robot robot_0(0, Point(-0.5, 0.2), Vector(), Angle::zero(), AngularVelocity::zero(),
-                  Timestamp::fromSeconds(0));
-    // default is all capabilities, if not specified otherwise
-    Robot robot_1(1, Point(-0.5, -0.2), Vector(), Angle::zero(), AngularVelocity::zero(),
-                  Timestamp::fromSeconds(0));
-    friendly_team.updateRobots({robot_0, robot_1});
-
-    friendly_team.assignGoalie(0);
-    world.updateFriendlyTeamState(friendly_team);
-
-    auto asst = stp.assignRobotsToTactics({}, world, true);
-
-    EXPECT_EQ(1, asst.size());
-    for (const auto& [tactic, robot] : asst)
-    {
-        UNUSED(tactic);
-        EXPECT_EQ(robot, robot_0);
-    }
-
-    // Change the goalie and perform the same check in case we have a fluke bug
-    friendly_team.assignGoalie(1);
-    world.updateFriendlyTeamState(friendly_team);
-
-    asst = stp.assignRobotsToTactics({}, world, true);
-
-    EXPECT_EQ(1, asst.size());
-    for (const auto& [tactic, robot] : asst)
-    {
-        UNUSED(tactic);
-        EXPECT_EQ(robot, robot_1);
-    }
-}
-
-TEST_F(STPTacticAssignmentTest,
-       test_assigning_stop_tactics_to_unassigned_non_goalie_robots)
-{
-    // Test that StopTactic is assigned to remaining robots without tactics
-
-    Team friendly_team(Duration::fromSeconds(0));
-    Robot robot_0(0, Point(-1, 1), Vector(), Angle::zero(), AngularVelocity::zero(),
-                  Timestamp::fromSeconds(0));
-    Robot robot_1(1, Point(-3, 5), Vector(), Angle::zero(), AngularVelocity::zero(),
-                  Timestamp::fromSeconds(0));
-    Robot robot_2(2, Point(6, 7), Vector(), Angle::zero(), AngularVelocity::zero(),
-                  Timestamp::fromSeconds(0));
-    friendly_team.updateRobots({robot_0, robot_1, robot_2});
-    world.updateFriendlyTeamState(friendly_team);
-
-    auto move_tactic_1 = std::make_shared<MoveTactic>();
-
-    move_tactic_1->updateControlParams(Point(-1, 0), Angle::zero(), 0,
-                                       TbotsProto::MaxAllowedSpeedMode::PHYSICAL_LIMIT);
-
-    ConstTacticVector tactics                   = {move_tactic_1};
-    std::vector<Robot> expected_robots_assigned = {robot_0, robot_1, robot_2};
-
-    auto asst = stp.assignRobotsToTactics({tactics}, world, false);
-
-    // Check each tactic is assigned to the intended robot
-    for (unsigned int i = 0; i < tactics.size(); i++)
-    {
-        ASSERT_TRUE(asst.find(tactics[i]) != asst.end());
-        EXPECT_EQ(asst.find(tactics[i])->second, expected_robots_assigned[i]);
-    }
-}
-
-TEST_F(STPTacticAssignmentTest, test_greediness_of_tiered_assignment)
```

Comment:
why is this test not included in the new test file?

Reply:
I thought we discussed that since we are not using priority tactic list  any more in our current system, we will remove this test while porting to the new system